src/modules/sleep.ts

```diff
@@ -0,0 +1,25 @@
+import { ChatCommand, OnEvent } from 'purplet';
+
+const messages = [
+  'Shshshshsh friendo!! You have mentioned the myth!!!!',
+  'You dared to mention the myth?!?',
+  "What is that thing you just mentioned? I don't know what that is.",
+  'I think you meant "long nap"...',
+];
+
+export default ChatCommand({
```

Comment:
The response to saying sleep is pretty much just for mentions in normal chats, do we need the slash command for it at this earlyish stage of development?

Reply:
it is to test that slash commands actually are functional